osquery/core/posix/process.cpp

```diff
@@ -177,13 +177,21 @@ std::shared_ptr<PlatformProcess> PlatformProcess::launchPythonScript(
     const std::string& args) {
   std::shared_ptr<PlatformProcess> process;
   std::string argv;
-
-  if (!isPlatform(PlatformType::TYPE_FREEBSD)) {
-    argv = "/usr/local/osquery/bin/python " + args;
+  std::string osquery_path;
+
+  char const* tmp = getenv("OSQUERY_DEPS");
```

Comment:
Do you mind changing this logic to use our environment methods? They return an optional so you can check if the variable is initialized.

This method is really only used in the unit tests, we might want to refactor it out of PlatformProcess too.